src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form">
+          <label>
```

Comment:
Dont wrapp inputs with label. 
```suggestion
          <label for="surname">Surname:</label>
```
Add appropriate ids to form elements do the labels is connected

Reply:
I have corrected my code, please check now